cdk.json

```diff
@@ -17,12 +17,23 @@
     ]
   },
   "context": {
-    "cluster.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/cluster.json",
-    "kubelet.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/kubelet.json",
-    "namespaceworkloads.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/namespace-workloads.json",
-    "nodeexporter.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodeexporter-nodes.json",
-    "nodes.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodes.json",
-    "workloads.dashboard.url": "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/workloads.json",
+    "grafanaGitOpsConfig": {
+      "bootstrapRepo": {
+        "repoUrl": "https://github.com/aws-observability/aws-observability-accelerator",
+        "name": "grafana-dashboards",
+        "targetRevision": "main",
+        "path": "./artifacts/grafana-operator-manifests/eks/infrastructure"
+      },
+      "fluxTargetNamespace": "grafana-operator",
+      "bootstrapValues": {
+        "GRAFANA_CLUSTER_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/cluster.json",
```

Comment:
Is there a reason why this is an FQDN URL which happens to point to the same repo as the bootstrap repo? Is there a way to use the dashboard location based off the relative path in the repo and branch that is referenced here?

Reply:
urls are copied from the existing ones, no change, I guess you are asking for an improvement?
I dont think it is possible anyway, because the urls are used in the GrafanaDashboard manifest for the grafana operator, e.g. see https://github.com/freschri/aws-observability-accelerator/blob/java-pattern/artifacts/grafana-operator-manifests/eks/java/amg_grafana-dashboards.yaml